devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy

```diff
@@ -96,7 +97,35 @@ def poll() {
     refresh()
 }
 
-def configure() {
-    log.debug "Configuring Reporting and Bindings."
+/**
+ * PING is used by Device-Watch in attempt to reach the Device
+ * */
+def ping() {
+    return zigbee.levelRefresh()
+}
+
+def healthPoll() {
+    log.debug "healthPoll()"
+    def cmds = refresh()
+    cmds.each{ sendHubCommand(new physicalgraph.device.HubAction(it))}
+}
+
+def configureHealthCheck() {
+    Integer hcIntervalMinutes = 12
+    log.debug "Configuring Health Check, Reporting and Bindings."
+    unschedule("healthPoll")
+    runEvery5Minutes("healthPoll")
+    // Device-Watch allows 2 check-in misses from device
+    sendEvent(name: "checkInterval", value: 60 * hcIntervalMinutes, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
     zigbee.onOffConfig() + zigbee.levelConfig() + zigbee.onOffRefresh() + zigbee.levelRefresh()
 }
+
+def configure() {
+    log.debug "configure()"
+    configureHealthCheck()
```

Comment:
I see you added `configureHealthCheck()` to both configure() and updated(). Is this necessary? What was the thought behind this? Thanks